src/main/java/net/minestom/server/fluid/FluidImpl.java

```diff
@@ -0,0 +1,34 @@
+package net.minestom.server.fluid;
+
+import net.minestom.server.registry.Registry;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public record FluidImpl(Registry.FluidEntry registry, NamespaceID namespace, int id) implements Fluid, Fluids {
```

Comment:
Why does the `FluidImpl` record inherits from the `Fluids` interface which contains only the generated fluid types?